app/src/common/components/article/list.jsx

```diff
@@ -0,0 +1,40 @@
+import {connect} from 'react-redux';
+import {Component, PropTypes} from 'react';
+
+import {loadArticleList} from '../../actions/article-list';
+import {ArticleLine} from './line';
+
+/** Component that displays the list of all articles, connected to the store. */
+@connect(
```

Comment:
Can you user the `compose` function of redux and not eh annotation. And the function enhancement.